packages/client/ui/react-ui/src/providers/CrossmintAuthProvider.test.tsx

```diff
@@ -101,12 +96,13 @@ describe("CrossmintAuthProvider", () => {
             embeddedWallets,
         });
 
-        expect(getByTestId("status").textContent).toBe("in-progress");
+        expect(getByTestId("wallet-status").textContent).toBe("in-progress");
```

Comment:
should probably check the initial auth status here too

Reply:
done